systemtest/src/test/java/io/strimzi/systemtest/log/LogSettingST.java

```diff
@@ -182,181 +187,222 @@
+        LOGGER.info("Checking if Bridge has log level set properly");
+        assertThat("Bridge's log level is set properly", checkLoggersLevel(BRIDGE_LOGGERS, BRIDGE_MAP), is(true));
+        assertThat("Bridge's GC logging is enabled", checkGcLoggingDeployments(bridgeDepName), is(true));
 
-        assertThat("Connect GC logging is disabled", checkGcLoggingDeployments(KafkaConnectResources.deploymentName(CONNECT_NAME)), is(false));
-        assertThat("Mirror-maker GC logging is disabled", checkGcLoggingDeployments(KafkaMirrorMakerResources.deploymentName(MM_NAME)), is(false));
-        assertThat("Mirror-maker2 GC logging is disabled", checkGcLoggingDeployments(KafkaMirrorMaker2Resources.deploymentName(MM2_NAME)), is(false));
+        KafkaBridgeResource.replaceBridgeResource(BRIDGE_NAME, bridge -> bridge.getSpec().setJvmOptions(JVM_OPTIONS));
+        DeploymentUtils.waitTillDepHasRolled(bridgeDepName, 1, bridgePods);
+        assertThat("Bridge GC logging is disabled", checkGcLoggingDeployments(bridgeDepName), is(false));
 
-        if (cluster.isNotKubernetes()) {
-            assertThat("ConnectS2I GC logging is disabled", checkGcLoggingDeploymentConfig(KafkaConnectS2IResources.deploymentName(CONNECTS2I_NAME)), is(false));
-        }
+        kubectlGetStrimzi(BRIDGE_NAME);
+        checkContainersHaveProcessOneAsTini(BRIDGE_NAME);
     }
 
-    @Test
-    @Order(13)
-    void testKubectlGetStrimzi() {
-        String userName = "test-user";
-        String topicName = "test-topic";
-
-        KafkaTopicResource.topic(CLUSTER_NAME, topicName).done();
-        KafkaUserResource.tlsUser(CLUSTER_NAME, userName).done();
-
+    void kubectlGetStrimzi(String resourceName) {
+        LOGGER.info("Checking if kubectl get strimzi contains {}", resourceName);
         String strimziCRs = cmdKubeClient().execInCurrentNamespace("get", "strimzi").out();
-
-        assertThat(strimziCRs, containsString(CLUSTER_NAME));
-        assertThat(strimziCRs, containsString(GC_LOGGING_SET_NAME));
-        assertThat(strimziCRs, containsString(MM_NAME));
-        assertThat(strimziCRs, containsString(MM2_NAME));
-        assertThat(strimziCRs, containsString(BRIDGE_NAME));
-        assertThat(strimziCRs, containsString(CONNECT_NAME));
-        assertThat(strimziCRs, containsString(userName));
-        assertThat(strimziCRs, containsString(topicName));
+        assertThat(strimziCRs, containsString(resourceName));
     }
 
-    @Test
-    @Order(14)
-    void testCheckContainersHaveProcessOneAsTini() {
+    void checkContainersHaveProcessOneAsTini(String resourceClusterName) {
         //Used [/] in the grep command so that grep process does not return itself
         String command = "ps -ef | grep '[/]usr/bin/tini' | awk '{ print $2}'";
 
-        for (Pod pod : kubeClient().listPods()) {
+        for (Pod pod : kubeClient().listPods(Labels.STRIMZI_CLUSTER_LABEL, resourceClusterName)) {
```

Comment:
We had there some issue that from time to time some containers weren't ready. Did we solve it somehow? If not, we can maybe add some checks for container readiness.

Reply:
Well the problem was (hopefully) solved. But I will add the readiness check to be sure. Thanks